libraries/crypto/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = [
   "Fabian Kaczmarczyck <kaczmarczyck@google.com>",
   "Guillaume Endignoux <guillaumee@google.com>",
   "Jean-Michel Picod <jmichel@google.com>",
+  "Julien Cretin <cretin@google.com>",
```

Comment:
I wouldn't put my name here. I don't think I touched the crypto that much.

Reply:
As you wish.